Organic/Organic.php

```diff
@@ -287,12 +287,10 @@ public function init( $apiUrl = null, $cdnUrl = null ) {
         /* Load up our sub-page configs */
         new AdminSettings( $this );
         new CCPAPage( $this );
-        new AdsTxt( $this );
```

Comment:
Have you tested these changes?
The `AdsTxt` ads custom action to  WP `init` filter during the initialization:
https://github.com/orgnc/wordpress-plugin/blob/d878e613952990f212f426124be6e3a7a9369e68/Organic/AdsTxt.php#L22

So.. without initialization of `AdsTxt` all of this won't work I suppose

Reply:
Oh! I see, yeah. Didn't saw that 👍